actors/miner/src/lib.rs

```diff
@@ -1655,30 +1593,27 @@ impl Actor {
                     return Err(actor_error!(illegal_argument, "too many deals for sector {} > {}", precommit.deal_ids.len(), deal_count_max));
                 }
 
-                let computed_cid = &verify_return.unsealed_cids[i];
-
                 // 1. verify that precommit.unsealed_cid is correct
                 // 2. create a new on_chain_precommit
 
-                let commd = match precommit.unsealed_cid {
-                    // if the CommD is unknown, use CommD computed by the market
-                    None => CompactCommD::new(*computed_cid),
-                    Some(x) => x,
-                };
-                if commd.0 != *computed_cid {
-                    return Err(actor_error!(illegal_argument, "computed {:?} and passed {:?} CommDs not equal",
-                            computed_cid, commd));
+                let declared_commd = precommit.unsealed_cid.unwrap();
```

Comment:
Will this not be an issue during a migration period? Can there not be "null" pre-commits on-chain right now?

Reply:
This is checked as a precondition earlier in this method (the code block that your previous comment is attached to). But thanks - you've pointed out that we can remove this Option now. I'll do it in a follow-up.